app/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,9 @@
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE"></uses-permission>
```

Comment:
Please make your xml consistence.
Use "<uses-permission .... />" instead of "<uses-permission ....></uses-permission>"